src/components/client/donation-flow/DonationFlowForm.tsx

```diff
@@ -247,10 +247,7 @@ export function DonationFlowForm() {
               </Button>
               <Grid2 container size={{ xs: 12, md: 7 }} direction={'column'}>
                 <StepSplitter content="1" active={Boolean(values.amountChosen)} />
-                <Amount
-                  disabled={values.payment === DonationFormPaymentMethod.BANK}
-                  error={Boolean(errors.finalAmount) && Boolean(submitCount > 0)}
-                />
+                <Amount error={Boolean(errors.finalAmount) && Boolean(submitCount > 0)} />
```

Comment:
Can we please revert this change for now? 
I understand the reasoning for this, but since this Amount Picker has no effect on bank transactions, in my opinion it will cause more confusion to keep it enabled.
I've already contacted the design team to look at this. If there are any changes regarding this step, lets handle them in a separate PR.

Reply:
Yes, I just reverted this one. Also reverted the change in `Amount` validation to not take into account the bank type payment with this commit https://github.com/podkrepi-bg/frontend/pull/1962/commits/ef26ae382f30aa2ee2449d33615c90645c090128
Here is a visual what I meant that the user cannot select another amount when Bank is selected and the calculation at the bottom cannot be used if the user wants to see what the fees would be.
cc @swolf86

![chrome_WnBgUR0833](https://github.com/user-attachments/assets/44a1d82a-35f2-4ff2-9ad5-fae88d8ce168)

Reply:
@nikolay-yankov 
I think the transactions fees should not be displayed it should be ---- until we have the integration with paysera. 

Why the amount cannot be changed, after selecting a bank transfer?
I saw that too